tests/test_unit/test_SWC_Blueprint.py

```diff
@@ -0,0 +1,68 @@
+from pathlib import Path
+
+from rsp_vision.objects.SWC_Blueprint import (
+    SessionFolder,
+    SubjectFolder,
+)
+
+
+def test_SWC_Blueprint_Spec(tmp_path, blueprint_spec):
+    Path(blueprint_spec.path).mkdir(parents=True, exist_ok=True)
+
+    assert (tmp_path / "my_project").exists()
+
+
+def test_SubjectFolder(tmp_path, blueprint_spec, folder_naming_specs):
```

Comment:
I may be mistaken, but these tests only test the crearion form folder naming specs, not from table, right? Could you increase the coverage by also testing the table case?